File: src/MeanscriptCpp/core/MArgs.cpp
#include "MS.h"
namespace meanscriptcore {
using namespace meanscript;

MArgs::MArgs (ByteCode* _byteCode, StructDef* _structDef, int32_t _base)
{
	byteCode = _byteCode;
	structDef = _structDef;
	baseIndex = _base;
	valid = true;
}

} // namespace meanscript(core)

File: java/net/meanscript/core/MArgs.java
package net.meanscript.core;
import net.meanscript.java.*;
import net.meanscript.*;





public class MArgs extends MC {
 ByteCode byteCode;
 StructDef structDef;
 int baseIndex; // stack base where struct data start from
 boolean valid; // become invalid when stack changes

public MArgs (ByteCode _byteCode, StructDef _structDef, int _base)
{
	byteCode = _byteCode;
	structDef = _structDef;
	baseIndex = _base;
	valid = true;
}

}
